GeneratorInterface/Pythia8Interface/test/test_Pythia8ConcurrentGeneratorFilter_WZ_TuneCP5_13TeV-pythia8.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+curl -s -k https://cms-pdmv.cern.ch/mcm/public/restapi/requests/get_fragment/BTV-RunIISummer20UL17GEN-00002 --retry 3 --create-dirs -o ${LOCALTOP}/src/Configuration/GenProduction/python/BTV-RunIISummer20UL17GEN-00002-fragment.py
+
+cd ${LOCALTOP}/src
+scram b
+cd ../..
+
+cmsDriver.py Configuration/GenProduction/python/BTV-RunIISummer20UL17GEN-00002-fragment.py --python_filename test_BTV-RunIISummer20UL17GEN-00002_1_cfg.py --eventcontent RAWSIM --customise Configuration/DataProcessing/Utils.addMonitoring --datatier GEN --fileout file:test_BTV-RunIISummer20UL17GEN-00002.root --conditions auto:run2_mc --beamspot Realistic25ns13TeVEarly2017Collision --customise_commands process.source.numberEventsInLuminosityBlock="cms.untracked.uint32(100)" --step GEN --geometry DB:Extended --era Run2_2017 --no_exec --mc -n 10 --nThreads 2 --nConcurrentLumis 0
```

Comment:
Shouldn't the cmsDriver be in sync with the above script? There 100 events are produced and nThreads 4. Should we include a transition in lumisection as this is know to be problematic. I.e. produce a total of 100 events and 10 per lumi block? @colizz @qliphy @PhyM73 what about adding concurrent workflows to relval?

Reply:
Thanks for pointing this out. I didn't require this `cmsDriver` to run too many events since I thought the _unittest_ might need to be a quick test. But I am unsure about this, maybe it could be a more comprehensive test? If so, then running 100 events with 10 events per lumi block on 4 threads or even more would be preferred of course. : )

I did test 10000 events with this module, and I haven't seen any problem with transitions yet. Another way to do local tests is using the `_cfg.py` file I provided, i.e., 

```bash
cd CMSSW_12_0_X/src
cmsenv
cmsRun ${CMSSW_BASE}/src/GeneratorInterface/Core/test/ConcurrentGeneratorFilter_WZ_TuneCP5_13TeV-pythia8_cfg.py
```

This way would be more convenient since its parameters, e.g., _events number_, _lumiblocks_, and _threads_, could be modified easily, and it does not need to build each time. However, it may only work under CMSSW_12_0_X.